ui/oopsyraidsy/data/07-dt/trial/valigarmanda-ex.ts

```diff
@@ -0,0 +1,62 @@
+import ZoneId from '../../../../../resources/zone_id';
+import { OopsyData } from '../../../../../types/data';
+import { OopsyTriggerSet } from '../../../../../types/oopsy';
+
+export type Data = OopsyData;
+
+const triggerSet: OopsyTriggerSet<Data> = {
+  zoneId: ZoneId.WorqorLarDorExtreme,
+  damageWarn: {
+    // initial phase & ubiquitouos
+    'ValigarmandaEx Spikecicle 1': '8FF5', // curved cleave
+    'ValigarmandaEx Spikecicle 2': '8FF6', // curved cleave
+    'ValigarmandaEx Spikecicle 3': '8FF7', // curved cleave
+    'ValigarmandaEx Spikecicle 4': '8FF8', // curved cleave
+    'ValigarmandaEx Spikecicle 5': '8FF9', // curved cleave
+    'ValigarmandaEx Sphere Shatter': '995D', // ice boulder aoe
+    'ValigarmandaEx Susurrant Breath': '8FC8', // conal
+    'ValigarmandaEx Slithering Strike': '8FCC', // point-blank AOE
+    'ValigarmandaEx Strangling Coil': '8FD0', // donut
+    'ValigarmandaEx Scourge of Thunder': '8FEA', // lightning spreads
+    // fire + adds mini-phase
+    'ValigarmandaEx Volcanic Drop Big': '8FE6', // half-room lava puddle
+    'ValigarmandaEx Volcanic Drop Puddle': '8FE4', // small lava puddle
+    'ValigarmandaEx Mountain Fire Cleave': '901A', // all but a small safe wedge
```

Comment:
Here and elsewhere, are any listed abilities sourced from combatants other than Valigarmanda? If so, could you include their name? You can see an example at https://github.com/OverlayPlugin/cactbot/blob/2a2081a7f09eaa924c992b296e16b09251853f53/ui/oopsyraidsy/data/03-hw/trial/thordan-ex.ts#L12.